### ISSUE TITLE
Questionmark is not a valid character in environment variables

### ISSUE THREAD

**Issue by lenaschoenburg** [grammarly/omniconf#7]
Question marks are not allowed in environment variables but are valid keywords in clojure. We should escape question mark to something (maybe `_Q_`) and use that as the variable name.

**Comment by alexander-yakushev** [comment]
This is a good point. However, if we are to go this way I prefer making a comprehensive munging/unmunging suite for each config source (env, cli opts, etc.), not just limit ourselves to question mark. What do you think?

**Comment by lenaschoenburg** [comment]
Agreed. I wonder if the clojure source code contains some code for this already? For munging function names in exceptions etc.

**Comment by alexander-yakushev** [comment]
There is munging, but it is for making Clojure literals Java (JVM) friendly. We'll need custom munging/unmunging for our purposes.
